lucene/core/src/java/org/apache/lucene/search/Weight.java

```diff
@@ -174,6 +174,31 @@ public BulkScorer bulkScorer(LeafReaderContext context) throws IOException {
     return new DefaultBulkScorer(scorer);
   }
 
+  /**
+   * Counts the number of documents that match a given {@link Weight#parentQuery} in a leaf. The
```

Comment:
We might need to be explicit that deleted documents are ignored, since this is not the case for other Weight APIs like {{Weight#scorer}}.

```suggestion
   * Counts the number of live documents that match a given {@link Weight#parentQuery} in a leaf. The
```

Reply:
Sure. Fixed it in 3d876d0